flexmeasures/utils/app_utils.py

```diff
@@ -102,6 +105,63 @@ def set_secret_key(app, filename="secret_key"):
         sys.exit(2)
 
 
+def root_dispatcher():
+    """
+    Re-routes to root views fitting for the current user,
+    depending on the FLEXMEASURES_ROOT_VIEW setting.
+    """
+    default_root_view = root_view = "/dashboard"
+    root_view_configs = current_app.config.get("FLEXMEASURES_ROOT_VIEW", [])
+    if isinstance(root_view_configs, str):
+        root_view_configs = [root_view_configs]  # ignore: type
+    for root_view_config in root_view_configs:
+        root_view = parse_applicable_viewname(
+            root_view_config, "FLEXMEASURES_ROOT_VIEW"
+        )
+        if root_view is not None:
+            break
+    if not root_view.startswith("/"):
```

Comment:
By not setting a default (non-account-role dependent) root, I managed to get a `root_view = None`, which causes this error:
```
AttributeError: 'NoneType' object has no attribute 'startswith'
```
We should probably swap lines 123-124 with 125-126?